Monika After Story/game/script-stories.rpy

```diff
@@ -348,11 +412,556 @@ label mas_story_immortal_love:
+    m 1eud "So there the man stood, weary eyed, among the animals all night long as the women circled around the horses, trying to find a way to get to the man."
+    m 1esa "Just before dawn, the women gave up and retreated back into the woods."
+    m 1esd "Now alone, the man cautiously headed back towards the hunting lodge, hearing no sound from within."
+    # chance of 1 in 14
+    if renpy.random.randint(1,14) == 1 or mas_full_scares:
+        play sound "sfx/stab.ogg"
+        show blood splatter1 as bl2 zorder 16:
+            pos (50,95)
+        show blood splatter1 as bl3 zorder 16:
+            pos (170,695)
+        show blood splatter1 as bl4 zorder 16:
+            pos (150,395)
+        show blood splatter1 as bl5 zorder 16:
+            pos (950,505)
+        show blood splatter1 as bl6 zorder 16:
+            pos (700,795)
+        show blood splatter1 as bl7 zorder 16:
+            pos (1050,95)
+        $ pause(1.5)
+        stop sound
+        hide bl2
+        hide bl3
+        hide bl4
+        hide bl5
+        hide bl6
+        hide bl7
+    m 3wuo "When he looked inside, he saw his three comrades dead on the floor, their skin almost translucent, as they lay in a pool of their own blood."
+    call mas_scary_story_cleanup
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_scary_story_serial_killer",
+    category=[store.mas_stories.TYPE_SCARY], prompt="The serial killer",unlocked=False),
+    eventdb=store.mas_stories.story_database)
+
+label mas_scary_story_serial_killer:
+    call mas_scary_story_setup
+    m 4esb "A young couple park at a cemetery one night for some undisturbed lovemaking."
+    m 4wud "They are interrupted by a radio report that a mass murderer has escaped from a psychiatric hospital nearby and may be headed in their direction."
+    m "They decide to leave, but the car won't start."
+    m 4esd "The young man gets out of the car to go for help and instructs the girl to stay in the car with the doors locked."
+    m "A few moments later, she hears a scratching sound on the roof of the car."
+    m 1esd "She thinks to herself it must be a tree branch in the wind."
+    m 1eud "Her date doesn't return and after some time passes, a police car drives by."
+    m "An officer stops and gets out of the car."
+    m 3eud "The police officer instructs the girl to get out of the car, walk towards him and not look back."
+    m "She does as he instructs her to. Not too long after, more police cars arrive."
+    m "Curiosity gets the better of the girl and she looks behind her."
+    m 4wuo "She is horrified to see the body of her boyfriend hanging head down from a tree, his throat slit from ear to ear."
+    # chance of 1 in 8
+    if renpy.random.randint(1,8) == 1 or mas_full_scares:
+        show y_sticker hopg zorder 17:
+            pos(600,425)
+            alpha 1.0
+            linear 1.0 alpha 0

```

Comment:
this linear is too short. should try to make this so the sound ends as yuri sticker disappears

Reply:
sure